src/alire/os_linux/alire-platform.adb

```diff
@@ -41,15 +41,30 @@ package body Alire.Platform is
                   Normalized : constant String :=
                                  To_Lower_Case (Replace (Line, " ", ""));
                begin
-                  if Starts_With (Normalized, "id=") then
-                     Cached_Distro :=
-                       Platforms.Distributions'Value (Tail (Normalized, '='));
-                     Distro_Cached := True;
-                     return Cached_Distro;
-                  end if;
-               exception
-                  when others =>
-                     exit; -- Not a known distro.
+                  begin
+                     if Starts_With (Normalized, "id=") then
```

Comment:
Better that repeating the whole block, I'd prefer to have 

```suggestion
                     if Starts_With (Normalized, "id=") or else Starts_With (Normalized, "id_like=") then
```
since the rest of the code is common, if my eyes are not missing anything.